templates/sdk-github-issue-template.yaml

```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: input
+    id: library-version
+    attributes:
+      label: Library version
+      description: What version of the library are you using?
+      placeholder: ex. v1.2.21
+    validations:
+      required: true
+  # Can remove this field for most SDKs, but useful for JS
```

Comment:
Honestly I think this wouldn't even be useful for JS (and certainly not for js v2 😄). I would suggest replacing this with a "Which platform are you experiencing this issue on" (runtime version, browser/react-native/nodejs, etc) which would probably be useful for every SDK.